booklog/docs/REDIS_INSTALLATION_GUIDE.md

```
@@ -0,0 +1,184 @@
+    redis:
+      host: localhost
+      port: 6379
+```
+
+### 2. 애플리케이션 실행
+```powershell
+.\gradlew bootRun
+```
+
+### 3. 로그 확인
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)